samples/ortest-ec2.ob

```diff
@@ -22,6 +22,7 @@ use_lua_resty_core = 0;
 use_stap = 0;
 nginx_cc_opt = '-O1 -I/opt/ssl/include';
 nginx_ld_opt = '-L/opt/ssl/lib';
+luajit_xcflags = '-DLUAJIT_ENABLE_GC64 -DLUAJIT_ENABLE_LUA52COMPAT -msse4.2';
```

Comment:
`-DLUAJIT_NUMMODE=2 -DLUA_USE_APICHECK -DLUA_USE_ASSERT` should be moved here as well.

Reply:
We don't use those in the openresty-debug target. I will add them, but just making sure this is what we want :)

Reply:
Updated!

Reply:
@thibaultcha Oh, yeah, openresty's `--with-debug` automatically enables those already, but it doesn't hurt to duplicate those flags anyway :)